screengrab/screengrab-lib/build.gradle

```diff
@@ -135,12 +141,15 @@ afterEvaluate {
 
 /*
  * Requires the following settings in ~/.gradle/gradle.properties
+ * signing.enabled=true
  * signing.keyId=516466E9 # last 8 digits of signing key (can get with `gpg -K`) 
  * signing.password=
  * signing.secretKeyRingFile=/Users/josh/.gnupg/secring.gpg # export with `gpg --keyring secring.gpg --export-secret-keys > ~/.gnupg/secring.gpg`
  */
-signing {
-    sign publishing.publications
+if (new Boolean(project.findProperty('signing.enabled'))) {
+    signing {
+        sign publishing.publications
+    }
```

Comment:
@penn5 This `signing` was added for the release process of the library to maven central. This doesn't effect anything with the users using this library, right? This will _only_  get used when I'm building/releasing a new version of the lib? 😇

Reply:
Correct, but it _is_ used by publishToMavenLocal which I needed when testing it

Reply:
Generally I find a lot of libraries won't build unless you supply signing keys and it makes me sad, so I always fix it while making my PR.

Reply:
Ohhhhhhhh, I forgot about `publishToMavenLocal` 🤦‍♂️ Thank you for fixing this then! Just wanted to make sure I fully understood the change 😊